src/app/app.config.ts

```diff
@@ -24,7 +25,7 @@ export const appConfig: ApplicationConfig = {
           deps: [HttpClient],
         },
       })
-    ),
+    ), provideAnimationsAsync(),
```

Comment:
Melhoria, não terminar com virgulas no último elemento
```suggestion
    ), provideAnimationsAsync()
```

Reply:
Thanks, bom olho 👍 . No futuro quero implementar uma regra que ao fazer push verifica se a indentação está correcta.